modules/svg/config.h

```diff
@@ -1,33 +0,0 @@
-/*************************************************************************/
-/*  config.h                                                             */
```

Comment:
This was bogus. ThorVG does require a `config.h` but it's better to give it a proper one - I've done that in `thirdparty/thorvg/inc/config.h`.